server/live-loan-router.js

```diff
@@ -0,0 +1,125 @@
+const get = require('lodash/get');
+const express = require('express');
+const argv = require('./util/argv');
+const config = require('../config/selectConfig')(argv.config);
+const fetch = require('./util/fetch');
+
+const drawLoanCard = require('./util/live-loan/live-loan-draw');
+
+function getRecommendedLoans(type, loginId, offset = 1) {
+	return new Promise((resolve, reject) => {
+		const endpoint = config.app.graphqlUri;
+		let query;
+		if (type === 'url') {
+			query = `{
+				ml {
+					recommendationsByLoginId(
+						segment: all
+						  loginId: ${loginId}
+						  offset: ${offset}
+						  limit: ${limit}
+					) {
+						values {
+							id,
+						}
+					}
+				}
+			}`;
+		} else if (type === 'img') {
+			query = `{
+				ml {
+					recommendationsByLoginId(
+						segment: all
+						  loginId: ${loginId}
+						  offset: 0
+						  limit: ${offset}
+					) {
+						values {
+							name
+							id
+							borrowerCount
+							geocode {
+								country {
+									name
+								}
+							}
+							use
+							loanAmount
+							status
+							loanFundraisingInfo {
+								fundedAmount
+							}
+							image {
+								retina: url(customSize: "w960h720")
+							}
+						}
+					}
+				}
+			}`;
+		}
+
+		fetch(endpoint, {
+			method: 'POST',
+			headers: { 'Content-Type': 'application/json' },
+			body: JSON.stringify({
+				query
+			}),
+		})
+			.then(result => result.json())
+			.then(result => {
+				const loanData = get(result, 'data.ml.recommendationsByLoginId.values');
+				resolve(loanData);
+			}).catch(err => {
+				console.error(err);
+				reject(err);
+			});
+	});
+}
+
+module.exports = function liveLoanRouter() {
+	const router = express.Router();
+	const userRouter = express.Router({ mergeParams: true });
+	const urlRouter = express.Router({ mergeParams: true });
+	const imgRouter = express.Router({ mergeParams: true });
+
+	router.get('/', (req, res) => { res.sendStatus(500); });
```

Comment:
500 should only be sent in genuine server failure cases. For these routes that you aren't doing anything except sending the 500 status, I would recommend just not including a handler at all. That will let the route fall through to the generic 404.

Reply:
> 500 should only be sent in genuine server failure cases. For these routes that you aren't doing anything except sending the 500 status, I would recommend just not including a handler at all. That will let the route fall through to the generic 404.

Nice, router section got a lot simpler from this feedback.